routing/providerquerymanager/providerquerymanager.go

```diff
@@ -20,9 +20,15 @@ import (
 var log = logging.Logger("routing/provqrymgr")
 
 const (
-	defaultMaxInProcessRequests = 16
-	defaultMaxProviders         = 0
-	defaultTimeout              = 10 * time.Second
+	// DefaultMaxInProcessRequests is the default maximum number of requests
+	// that are processed concurrently. A value of 0 means unlimited.
+	DefaultMaxInProcessRequests = 8
+	// DefaultMaxProviders is the default maximum number of providers that are
+	// looked up per find request. 0 value means unlimited.
+	DefaultMaxProviders = 0
+	// DefaultTimeout is the limit on the amount of time to spend waiting for
+	// the maximum number of providers from a find request.
+	DefaultTimeout = 10 * time.Second
```

Comment:
Do we still want to export these?

Reply:
I believe so – we should export default for every thing that we also allow being tweaked via `WithFoo` funcs.

Reply:
That is correct. We can add config values to configure the default pqm in a separate PR.